services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/rpc/_disk.py

```diff
@@ -7,5 +7,5 @@
 
 
 @router.expose()
-async def free_reserved_disk_space(_: FastAPI) -> None:
-    disk.remove_reserved_disk_space()
+async def delete_reserved_disk(_: FastAPI) -> None:
+    disk.delete_reserved_disk()
```

Comment:
this one I don't understand the renaming as I find it more confusing than ever.
I checked your code, and this actually removes a file that I believe has the size of the space you require.
I think the old name was better.,